index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
```

Comment:
There's some general indentation issues in this doc, which can easily be fixed by Atom's auto-indent feature. Check it out...